src/core/legacy_lexer.mll

```diff
@@ -24,7 +24,7 @@ let locate_lexbuf : Lexing.lexbuf -> Pos.pos = fun lexbuf ->
 
 type token =
   (* Special characters and symbols. *)
-  | L_SQB | R_SQB | L_PAR | R_PAR | ARROW | LARROW | FARROW | DEFEQ | COMMA
+  | L_SQB | R_SQB | L_PAR | R_PAR | LEFTBRA | RIGHTBRA | ARROW | LARROW | FARROW | DEFEQ | COMMA
```

Comment:
Why do you change the legacy parser? The legacy parser is a parser for the current version of Dedukti. It should not include things that are not in Dedukti.

Reply:
Well, in this case, this is harmless. So, that's fine.